<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerSettingsGeneralPanel</class>
 <widget class="ctkSettingsPanel" name="qSlicerSettingsGeneralPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="DefaultScenePathLabel">
     <property name="toolTip">
      <string>Directory where scenes are saved to by default</string>
     </property>
     <property name="text">
      <string>Default scene location:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ctkDirectoryButton" name="DefaultScenePathButton"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="ShowSplashScreenLabel">
     <property name="text">
      <string>Disable splash screen:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="ShowSplashScreenCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ctkLanguageComboBox" name="LanguageComboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="LanguageLabel">
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="ConfirmRestartLabel">
     <property name="text">
      <string>Confirm on restart:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="ConfirmRestartCheckBox">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="ConfirmExitLabel">
     <property name="text">
      <string>Confirm on exit:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="ConfirmExitCheckBox">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="SlicerWikiURLLabel">
     <property name="text">
      <string>Slicer Wiki URL:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="SlicerWikiURLLineEdit"/>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="NumOfRecentlyLoadedFiles">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="NumOfRecentlyLoadedFilesLabel">
     <property name="text">
      <string>Max. number of 'Recently Loaded' menu items:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkSettingsPanel</class>
   <extends>QWidget</extends>
   <header>ctkSettingsPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkLanguageComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkLanguageComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDirectoryButton</class>
   <extends>QWidget</extends>
   <header>ctkDirectoryButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
